examples/protocols/multicolor-particle-calibration/multicolor-particle-calibration.py

```diff
@@ -372,7 +371,7 @@
     "SerialDilution",
     source=fluorescein_wells_A1.output_pin("samples"),
     destination=dilution_series1.output_pin("samples"),
-    amount=sbol3.Measure(200, OM.microlitre),
+    amount=sbol3.Measure(100, OM.microlitre),
```

Comment:
Per the above comment, the way the Primitive is implemented, the `amount` is interpreted as the "final volume of the last container in the series".  Perhaps we should change the Pin name to reflect this.  Or alternatively, change the Pin name to reflect "transfer volume" instead of "final volume" so that the semantics are not ambiguous